pkg/cmd/render/render_test.go

```diff
@@ -274,6 +315,23 @@ func TestTemplateDataIpv4(t *testing.T) {
 	testTemplateData(config)
 }
 
+func TestRenderScalingStrategyBootstrapInPlace(t *testing.T) {
+	want := TemplateData{
+		BootstrapScalingStrategy:  ceohelpers.BootstrapInPlaceStrategy,
+		EtcdEndpointConfigmapData: "MTAuMC4wLjE: 10.0.0.1",
```

Comment:
(No action required) What's the logic behind these obfuscated keys?

Reply:
it was an implementation detail with the endpoint controller functionally the idea is to generate a unique key by base64 encoding the IP.